usr/share/rear/output/ISO/Linux-i386/250_populate_efibootimg.sh

```diff
@@ -137,7 +137,7 @@ cp $v -r $TMP_DIR/mnt/EFI $TMP_DIR/isofs/ || Error "Could not create the isofs/E
 # Make /boot/grub/grub.cfg available on isofs/
 mkdir $v -p -m 755 $TMP_DIR/isofs/boot/grub
 if test "$( type -p grub )" ; then
-    cp $v $TMP_DIR/isofs/EFI/BOOT/BOOTX64.conf $TMP_DIR/isofs/boot/grub/ || Error "Could not copy EFI/BOOT/BOOTX64.conf to isofs/boot/grub"
+    cp $v $TMP_DIR/isofs/EFI/BOOT/BOOT$EFI_ARCH_UPPER.conf $TMP_DIR/isofs/boot/grub/ || Error "Could not copy EFI/BOOT/BOOT${EFI_ARCH_UPPER}.conf to isofs/boot/grub"
```

Comment:
Same as
https://github.com/rear/rear/pull/3157/files#r1511412487

```
BOOT$EFI_ARCH_UPPER.conf
```
should work